meta/main.yml

```diff
@@ -10,9 +10,11 @@ galaxy_info:
       - all
 
 dependencies:
- - role: ANXS.python
+ - role: DavisRayM.python
```

Comment:
Why do we need to switch?

Reply:
The `ANXS.python` ansible role doesn't use python3 to install pip (Doesn't install python3 too). It uses whatever `/usr/bin/python` links to which is usually python2 and this causes a few issues....

The `ANXS.python` role also doesn't specify the pip executable to use(defaulting to pip3 if python3 is present on the server) but the role installed pip2 which ends up failing unless you switch `ansible_python_interpreter` to `/usr/bin/python3` when using this playbook on a server with python3 installed